app/tree/index.html

```diff
@@ -6,7 +6,7 @@
       (function() {
         window.location.href = window.location.href.replace(
           /\/tree(\/|\/index.html)?$/,
-          '/../classic/tree/index.html'
+          '/classic/tree/index.html'
```

Comment:
this was the bug injected at some point re https://github.com/jtpio/jupyterlite/pull/75#issuecomment-827715152